### DISCUSSION TITLE
Add achievements roadmap and achievement guides

### DISCUSSION THREAD

**Discussion by anegoda1995** [RetroAchievements/RAWeb#1667] [Ideas]
I think it will be very helpful to have opportunity for creating achievements roadmap for whole game set like [here](https://psnprofiles.com/guide/15438-orgarhythm-trophy-guide#3-tips-strategies) and create guide per achievement like [here](https://psnprofiles.com/trophy/1559-orgarhythm/51-cinema-enthusiast).

My vision of it below
For every game will appear "Roadmap" button in main game sidebar, that will follow to https://retroachievements.org/game/1446/roadmap that will looks like forum "create topic" page or whatever WYSIWYG editor.
<img width="472" alt="image" src="https://github.com/RetroAchievements/RAWeb/assets/10788396/23ed8dda-46b1-44f0-8d91-6490135606cc">
And for every achievement page available list of guides
<img width="830" alt="image" src="https://github.com/RetroAchievements/RAWeb/assets/10788396/1b504d3d-c131-49f4-926a-41ac3579f629">

One big problem that I see – need a lot of space for storing images.
Does someone have ideas for code/ui design for these 2 features?

**Comment by FalcusDev** [comment]
What would the roadmap be for? Would that be like roadmaps that are used for achievement guides on other platforms? If that's the case, that's a fairly common format within the RAGuides repo, where it will have a roadmap along with the achievements but I feel that's something more to be included within the guides themselves.

**Reply by anegoda1995** [reply]
What it needs to be in my imagination, you can see the good example here: https://psnprofiles.com/guide/15438-orgarhythm-trophy-guide#3-tips-strategies

Roadmaps it is simple but detailed **roadmaps** :)
In roadmap you can see a path how to obtain achievements one-by-one for obtain 100% in shortest time if you want it.
For example in guides you can see in what act/stage/mission/level you need to pay attention for obtain missable achievement and do not replay game from the beginning. Also there you can see **link** to achievements where you can see more detailed description and tips (tips it is achievements guides – my second proposal in this topic).

Also it will be good to move it from RAGuides to the site

**Comment by suspect15** [comment]
I think this is pretty much already covered by the guides themselves.  I don't see much of a distinction from what you describe.  Perhaps what are looking for would be more appropriate as an RAGuide standard policy.

**Comment by wescopeland** [comment]
Superseded by https://github.com/RetroAchievements/RAWeb/discussions/4196.
